AlexMarrinan/RavenGuard
# [RGT-40] basic chest spawns

- Chests spawn at tiles defined in the chest layer of the level editor
- Spawns are pure random, the rate defined by GridManager.chestSpawnRate
- Chests will open and eventually despawn after a player unit runs over the assigned tile

Testing:
1. Set the GridManger's Test Mode to true. In test mode, it reads LevelBases from the TestBases folder
2. Run the game and the multiple chests should spawn on the board (default to 30% chance per chest tile)
3. Move any player units over the chests to "collect" them, for now just opening and despawning with no items 
4. Try adding chests to the other bases, switch Test Made back to false to go back to loading the standard levels
5. Try changing the chestSpawnRate value in GridManager
